src/core/components/choice-card/ChoiceCardGroup.stories.tsx

```diff
@@ -0,0 +1,317 @@
+import { ThemeProvider } from '@emotion/react';
+import { choiceCardDefault } from '@guardian/src-foundations/themes';
+import React, { useState } from 'react';
+import { Story } from '../../../@types/storybook-emotion-10-fixes';
+import { asChromaticStory, asPlayground } from '../../../lib/story-intents';
+import { ChoiceCard } from './ChoiceCard';
+import ChoiceCardStories from './ChoiceCard.stories';
+import { ChoiceCardGroup, ChoiceCardGroupProps } from './ChoiceCardGroup';
+
+export default {
+	title: 'Source/src-choice-card/ChoiceCardGroup',
+	component: ChoiceCardGroup,
+	subcomponents: { ChoiceCard },
+	args: {
+		name: 'colours',
+		label: 'Choose an option',
+		supporting: 'These are your options',
+		multi: false,
+		children: [
+			<ChoiceCard
+				{...ChoiceCardStories.args}
+				id="abc1"
+				value="option-1"
+				label="Option 1"
+				key={1}
+			/>,
+			<ChoiceCard
+				{...ChoiceCardStories.args}
+				id="abc2"
+				value="option-2"
+				label="Option 2"
+				key={2}
+			/>,
+			<ChoiceCard
+				{...ChoiceCardStories.args}
+				id="abc3"
+				value="option-3"
+				label="Option 3"
+				key={3}
+			/>,
+			<ChoiceCard
+				{...ChoiceCardStories.args}
+				id="abc4"
+				value="option-4"
+				label="Option 4"
+				key={4}
+			/>,
+			<ChoiceCard
+				{...ChoiceCardStories.args}
+				id="abc5"
+				value="option-5"
+				label="Option 5"
+				key={5}
+			/>,
+			<ChoiceCard
+				{...ChoiceCardStories.args}
+				id="abc6"
+				value="option-6"
+				label="Option 6"
+				key={6}
+			/>,
+		],
+	},
+	argTypes: {
+		columns: {
+			options: [undefined, 2, 3, 4, 5],
+			control: { type: 'select' },
+		},
+		error: {
+			options: [undefined, 'example error message'],
+			control: { type: 'select' },
+		},
+	},
```

Comment:
Can we disable the `cssOverrides` control as it doesn't work in storybook?

```suggestion
	argTypes: {
		columns: {
			options: [undefined, 2, 3, 4, 5],
			control: { type: 'select' },
		},
		error: {
			options: [undefined, 'example error message'],
			control: { type: 'select' },
		},
		cssOverrides: {
			control: { disable: true },
		},
	},
```

Reply:
as discussed offline, will do for all chromaticStories in separate PR